.gitignore

```diff
@@ -80,12 +80,14 @@ oprofile_data/
 /bin/bios
 /bin/dumps
 /bin/help
+/bin/settings
 /bin/inis
 /bin/logs
 /bin/memcards
 /bin/plugins
 /bin/snaps
 /bin/sstates
+/bin/portable*.yaml
```

Comment:
Why `portable*.yaml`? Shouldn't it just be `portable.yaml`?

Reply:
I implemented a workaround in visual studio to prevent the overwriting of the file.  If you have a `portable.build.yaml` in the `/bin` directory, the visual studio process will prefer that, over the gold-standard in the `/docs` directory.

This allows you to not have the `RunWizard` flipped back to `true` everytime you compile the executable, which was apparently required.  CMake does not currently have such a feature as far as i know.